packages/contracts/test/dollar/core/CreditNft.t.sol

```diff
@@ -27,7 +26,7 @@ contract CreditNftTest is LocalTestHelper {
         super.setUp();
 
         // deploy Credit NFT token
-        creditNft = new CreditNft(address(diamond));
+        creditNft = IUbiquityNft;
```

Comment:
can you explain this conversion?

Reply:
It seemed that across the tests it was more common to call functions using ```IContract.function()```,  other times addresses are being casted to the contract or interface when theres no reason not to use the the above. 

I will remove ```IUbiquityNFT``` and replace it with creditNft